docs/getting-started.md

```diff
@@ -9,11 +9,13 @@ These tools are easily deployed into your cluster with just one command: ```kube
 
 ## Installation of the Kubeapps Installer
```

Comment:
We changed the repo to kubeapps so I think we should call it `kubeapps cli` instead of `kubeapps installer`. It's growing up to more than just an "installer"